src/ui/setup/prefs.ts

```diff
@@ -159,7 +160,13 @@ async function maybeUpdateReplaySessions(state: UIState) {
     selectedPanel: getSelectedPanel(state),
     localNags: [...(previousReplaySession?.localNags || [])],
     tabs: persistTabs(getTabs(state)) || [],
+    pendingComment: selectors.getPendingComment(state),
   };
 
   asyncStore.replaySessions = { ...previousReplaySessions, [recordingId]: currentReplaySession };
 }
+
+export async function getReplaySession(recordingId: string) {
```

Comment:
we could use this in a few other places as well.

Reply:
I bet